raw-spans-grouper/raw-spans-grouper/src/main/java/org/hypertrace/core/rawspansgrouper/RawSpansGrouper.java

```diff
@@ -72,7 +76,17 @@ public StreamsBuilder buildTopology(
     streamsBuilder.addStateStore(spanStoreBuilder);
     streamsBuilder.addStateStore(traceStateStoreBuilder);
 
-    Produced<TraceIdentity, StructuredTrace> outputTopicProducer = Produced.with(null, null);
+    StreamPartitioner<TraceIdentity, StructuredTrace> groupPartitioner =
+        new GroupPartitionerBuilder<TraceIdentity, StructuredTrace>()
+            .buildPartitioner(
+                "spans",
```

Comment:
what is this first argument means - `spans` ? Is it a name? If so can we use `rawspans` as below we are already using `spans`

Reply:
spans is the profile name. these profiles must be defined in config service.
example profiles: spans, sessions, apis
traces traffic is same (ratio wise) as spans traffic. so, we are not planning to add traces profile initially.